sdk/core/azure-core/src/main/java/com/azure/core/util/tracing/TracerProxy.java

```diff
@@ -17,14 +17,13 @@
  */
 public final class TracerProxy {
 
-    private static final List<Tracer> TRACERS;
+    private static Tracer TRACER;
+
     static {
         ServiceLoader<Tracer> serviceLoader = ServiceLoader.load(Tracer.class);
-        List<Tracer> tracers = new ArrayList<>();
-        for (Tracer tracer : serviceLoader) {
-            tracers.add(tracer);
+        if (serviceLoader != null) {
+            TRACER = serviceLoader.iterator().next();
```

Comment:
Do we have any mechanisms available which will ensure a certain tracer is loaded if I happen to bring multiple? This can be a follow-up as it is definitely an edge case.

Reply:
Yeah, I think we could add a feature where a user preference could be preferred for a single implementation if multiple found.

Reply:
Or, if there are multiple instances of tracer implementations, should we support all of them? Create spans on each tracer implementation?

Reply:
There is no real use case to support multiple implementations, as users would always tie it with a single implementation of tracer to an exporter. 
It brings in a lot of complexity for the users of the library to actually look to configure each span with a single tracer implementation and then the corresponding exporter for that tracer. As mentioned above, not a real use case and something only Java SDK was supporting.